packages/gas-fee-controller/src/GasFeeController.test.ts

```diff
@@ -734,6 +734,12 @@ describe('GasFeeController', () => {
 
   describe('fetchGasFeeEstimates', () => {
     describe('when on any network supporting legacy gas estimation api', () => {
+      const getDefaultOptions = () => ({
```

Comment:
The reason why I placed this kind of setup in the test itself is because this information is critical to the test (where critical means that if we didn't have this data here, then the behavior we're testing wouldn't make sense, because this is what creates the scenario that sets up the test). In my experience, it's better in the long run to place this kind of data in the body of the test itself (in the function passed to `it`) so that it is easier to comprehend the test without having to necessarily look elsewhere. I can understand that you might want to move this closer to the `describe` so that it is easier to cross-check with the description there, but if the `describe` ends up being long then you end up having to go back and forth between this and the test itself to mentally construct the data that is being built.

That said I realize everyone has different reading styles and needs so I would definitely welcome your opinion here.